src/components/LocalIdentityModal/LocalIdentityModal.js

```diff
@@ -27,11 +22,11 @@ LocalIdentityModal.propTypes = {
 
 const Modal = ({ address, label, onCancel, onSave }) => {
   const [action, setAction] = React.useState()
```

Comment:
Not related to this PR, but I think it could be a good practice to always set a default for `useState()`, even if we want `undefined`.

Reply:
Fixed in 5b78370.